platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -154,62 +154,55 @@ of capabilities, platforms make it easy for their users to deliver valuable prod
-
-To be able to use a platform's offerings, users require documentation and
-examples \[4\]. So a platform team needs to not only make a capability like
-observability available, but also publish templates and docs to help users
-consume it. For example, a platform team might publish a reusable workflow for
-building, testing, deploying and verifying a Web application on Kubernetes.
-Beyond examples for individual resource types, advanced platforms should group
-collections of templates and guidance for specific scenarios, a bundle often
-described as a golden path.
-
-The capabilities provided by a platform should be available to users on demand
-\[5\] with minimal manual intervention. The interfaces discussed above - APIs and
-web pages - should enable users to get resources they need in a few minutes with
-at most an approval. For example, a web page describing a database management
-service should include a button to request a database and share a locator and
-credentials for that database within a few minutes of the request.
-
-One of the purposes of a platform is to reduce cognitive load on product teams,
-so services offered by the platform should not need to be operated by consumers;
-either the platform provider or its delegates, such as a cloud service, should
-handle those operations, hiding complexity \[7\] from users. For example, users
-may often require a message queue, but shouldn't have to manage a message
-broker.
-
-Platforms are intended to make product development more efficient, so it's
-important they not be an impediment \[8\] when the most efficient path is to not
-use the platform's offerings. Platforms should enable product teams to
-provide and manage their own capabilities when necessary. For example, if a
-platform team doesn't provide a graph database and it's required for a product,
-it should be possible for that product team to provision and operate a graph
-database themselves.
+After defining what a platform is and why an organization might want to build one,
+it's relevant to identify what are the key attributes that affect the success
+of a platform.
+
+1. **Platform as a product**. A platform exists to serve the requirements of its users
+   and it should be designed and evolved based on those requirements, similar to any
+   other software products.
+1. **Common requirements**. A platform should provide the necessary capabilities to
+   support the most common use cases across stream-aligned teams, and prioritize those
+   over more specific capabilities that are only used by a single team to maximize
+   the value delivered.
+1. **User experience**. A platform should offer its capabilities through consistent
+   interfaces and focus on the user experience. Users should be able to request, manage,
+   and observe such capabilities using a combination of GUIs, APIs, command-line tools,
+   IDEs integrations, and portals, depending on their needs. For example, a platform
```

Comment:
> Users should be able to request, manage,
> and observe such capabilities using a combination of GUIs, APIs, command-line tools,
> IDEs integrations, and portals, depending on their needs

This seems to challenge our points about thinnest viable platform, though I think the intention is spot on. I wonder we can shorten to just the first sentence? Or if we can some how couch the remaining explanation as a bigger endeavour than some platforms need?

Reply:
I agree. I think the first sentence is enough to get the point across, while listing everything one could build for a better DX can quickly become bloated.

Reply:
Should I keep the first sentence, remove the second one, and then leave the example? Or would you prefer leaving the example off as well?

Reply:
I think the last sentence is great where it brings to life the concept of meeting platform users where they are. Maybe a trimmed down version would be:

> 1. **User experience**. A platform should offer its capabilities through consistent
   interfaces and focus on the user experience. Platforms should endeavor to meet their users where they are which may mean different interfaces such as IDEs; command-line tools, chat bots, or web portals.

Reply:
+1 to the change and something like Abby's refactor, but Abby I'm not sure if you're suggesting removing the following sentence with an example or not? I'm okay with keeping it, it gives a bit of color; but okay with removing it too if we feel it's repetitive.

Reply:
I have rephrased the paragraph based on your suggestions. What do you think?

Reply:
love it Thomas!